vscode/package.json

```diff
@@ -57,11 +57,36 @@
 				"path": "./syntaxes/pest.tmLanguage.json"
 			}
 		],
+		"configuration": [
+			{
+				"id": "pestIdeTools",
+				"title": "Pest IDE Tools",
+				"properties": {
+					"pestIdeTools.serverPath": {
+						"type": "string",
+						"description": "Set a custom path to a Pest LS binary.",
+						"scope": "window"
+					},
+					"pestIdeTools.checkForUpdates": {
+						"type": "boolean",
+						"description": "Check for updates to the Pest LS binary.",
+						"default": false,
+						"scope": "window"
+					},
+					"pestIdeTools.alwaysUsedRuleNames": {
+						"type": "array",
+						"description": "Rule names that should not be included in the unused rule names diagnostic.",
+						"default": [],
+						"scope": "window"
+					}
+				}
+			}
+		],
 		"commands": [
 			{
-				"command": "pest-language-support.restartServer",
-				"title": "Restart Pest Language Server",
-				"category": "Pest Language Support"
+				"category": "Pest",
+				"command": "pestIdeTools.restartServer",
+				"title": "Restart Server"
```

Comment:
`Restart Pest Server` or `Restart Pest LSP`.

This will list in VS Code command list, when we search `restart` it will show many commands contains `restart`, so need difference in name.

Reply:
It actually shows up as `Pest: Restart Server` since it's in the Pest section, and you can search by 'pest'